extensions/go/src/settings.ts

```diff
@@ -23,11 +23,11 @@ export interface Settings {
      */
     'basicCodeIntel.unindexedSearchTimeout'?: number
     /**
-     * The WebSocket language server to connect to
+     * The address of the WebSocket language server to connect to.
```

Comment:
Maybe give an example for format? `wss://www.example.com/socketserver`  or `host:port` or whatever.

Reply:
Updated in both configs, regenerated settings files.